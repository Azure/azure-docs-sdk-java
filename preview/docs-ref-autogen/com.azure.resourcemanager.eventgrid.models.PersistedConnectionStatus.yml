### YamlMime:JavaType
uid: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus"
fullName: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus"
name: "PersistedConnectionStatus"
nameWithType: "PersistedConnectionStatus"
summary: "Defines values for PersistedConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PersistedConnectionStatus extends ExpandableStringEnum<PersistedConnectionStatus>"
constructors:
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.PersistedConnectionStatus()"
fields:
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.APPROVED"
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.PENDING"
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.eventgrid.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventgrid:1.0.0-beta.1
