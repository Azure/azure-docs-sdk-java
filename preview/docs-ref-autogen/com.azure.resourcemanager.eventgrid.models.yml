### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.eventgrid.models"
fullName: "com.azure.resourcemanager.eventgrid.models"
name: "com.azure.resourcemanager.eventgrid.models"
summary: "Package containing the data models for EventGridManagementClient. Azure EventGrid Management Client."
classes:
- "com.azure.resourcemanager.eventgrid.models.AdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.AdvancedFilterOperatorType"
- "com.azure.resourcemanager.eventgrid.models.AzureFunctionEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.BoolEqualsAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.ConnectionState"
- "com.azure.resourcemanager.eventgrid.models.DeadLetterDestination"
- "com.azure.resourcemanager.eventgrid.models.DeadLetterEndPointType"
- "com.azure.resourcemanager.eventgrid.models.DomainProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.DomainRegenerateKeyRequest"
- "com.azure.resourcemanager.eventgrid.models.DomainTopicProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.DomainTopicsListResult"
- "com.azure.resourcemanager.eventgrid.models.DomainUpdateParameters"
- "com.azure.resourcemanager.eventgrid.models.DomainsListResult"
- "com.azure.resourcemanager.eventgrid.models.EndpointType"
- "com.azure.resourcemanager.eventgrid.models.EventDeliverySchema"
- "com.azure.resourcemanager.eventgrid.models.EventHubEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionFilter"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionUpdateParameters"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionsListResult"
- "com.azure.resourcemanager.eventgrid.models.EventTypesListResult"
- "com.azure.resourcemanager.eventgrid.models.HybridConnectionEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.InboundIpRule"
- "com.azure.resourcemanager.eventgrid.models.InputSchema"
- "com.azure.resourcemanager.eventgrid.models.InputSchemaMapping"
- "com.azure.resourcemanager.eventgrid.models.InputSchemaMappingType"
- "com.azure.resourcemanager.eventgrid.models.IpActionType"
- "com.azure.resourcemanager.eventgrid.models.JsonField"
- "com.azure.resourcemanager.eventgrid.models.JsonFieldWithDefault"
- "com.azure.resourcemanager.eventgrid.models.JsonInputSchemaMapping"
- "com.azure.resourcemanager.eventgrid.models.NumberGreaterThanAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.NumberGreaterThanOrEqualsAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.NumberInAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.NumberLessThanAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.NumberLessThanOrEqualsAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.NumberNotInAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.OperationInfo"
- "com.azure.resourcemanager.eventgrid.models.OperationsListResult"
- "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus"
- "com.azure.resourcemanager.eventgrid.models.PrivateEndpoint"
- "com.azure.resourcemanager.eventgrid.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.eventgrid.models.PrivateEndpointConnectionsParentType"
- "com.azure.resourcemanager.eventgrid.models.PrivateLinkResourcesListResult"
- "com.azure.resourcemanager.eventgrid.models.PublicNetworkAccess"
- "com.azure.resourcemanager.eventgrid.models.ResourceProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.ResourceRegionType"
- "com.azure.resourcemanager.eventgrid.models.RetryPolicy"
- "com.azure.resourcemanager.eventgrid.models.ServiceBusQueueEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.ServiceBusTopicEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.StorageBlobDeadLetterDestination"
- "com.azure.resourcemanager.eventgrid.models.StorageQueueEventSubscriptionDestination"
- "com.azure.resourcemanager.eventgrid.models.StringBeginsWithAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.StringContainsAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.StringEndsWithAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.StringInAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.StringNotInAdvancedFilter"
- "com.azure.resourcemanager.eventgrid.models.TopicProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.TopicRegenerateKeyRequest"
- "com.azure.resourcemanager.eventgrid.models.TopicTypeProvisioningState"
- "com.azure.resourcemanager.eventgrid.models.TopicTypesListResult"
- "com.azure.resourcemanager.eventgrid.models.TopicUpdateParameters"
- "com.azure.resourcemanager.eventgrid.models.TopicsListResult"
- "com.azure.resourcemanager.eventgrid.models.WebhookEventSubscriptionDestination"
interfaces:
- "com.azure.resourcemanager.eventgrid.models.Domain"
- "com.azure.resourcemanager.eventgrid.models.Domain.Definition"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithInboundIpRules"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithInputSchema"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithInputSchemaMapping"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithPrivateEndpointConnections"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.eventgrid.models.Domain.DefinitionStages.WithTags"
- "com.azure.resourcemanager.eventgrid.models.Domain.Update"
- "com.azure.resourcemanager.eventgrid.models.Domain.UpdateStages"
- "com.azure.resourcemanager.eventgrid.models.Domain.UpdateStages.WithInboundIpRules"
- "com.azure.resourcemanager.eventgrid.models.Domain.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.eventgrid.models.Domain.UpdateStages.WithTags"
- "com.azure.resourcemanager.eventgrid.models.DomainSharedAccessKeys"
- "com.azure.resourcemanager.eventgrid.models.DomainTopic"
- "com.azure.resourcemanager.eventgrid.models.DomainTopics"
- "com.azure.resourcemanager.eventgrid.models.Domains"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.Definition"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithDeadLetterDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithEventDeliverySchema"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithExpirationTimeUtc"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithFilter"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithLabels"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithRetryPolicy"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.DefinitionStages.WithScope"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.Update"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithDeadLetterDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithDestination"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithEventDeliverySchema"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithExpirationTimeUtc"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithFilter"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithLabels"
- "com.azure.resourcemanager.eventgrid.models.EventSubscription.UpdateStages.WithRetryPolicy"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptionFullUrl"
- "com.azure.resourcemanager.eventgrid.models.EventSubscriptions"
- "com.azure.resourcemanager.eventgrid.models.EventType"
- "com.azure.resourcemanager.eventgrid.models.Operation"
- "com.azure.resourcemanager.eventgrid.models.Operations"
- "com.azure.resourcemanager.eventgrid.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.eventgrid.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.eventgrid.models.PrivateLinkResource"
- "com.azure.resourcemanager.eventgrid.models.PrivateLinkResources"
- "com.azure.resourcemanager.eventgrid.models.Topic"
- "com.azure.resourcemanager.eventgrid.models.Topic.Definition"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithInboundIpRules"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithInputSchema"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithInputSchemaMapping"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithPrivateEndpointConnections"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.eventgrid.models.Topic.DefinitionStages.WithTags"
- "com.azure.resourcemanager.eventgrid.models.Topic.Update"
- "com.azure.resourcemanager.eventgrid.models.Topic.UpdateStages"
- "com.azure.resourcemanager.eventgrid.models.Topic.UpdateStages.WithInboundIpRules"
- "com.azure.resourcemanager.eventgrid.models.Topic.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.eventgrid.models.Topic.UpdateStages.WithTags"
- "com.azure.resourcemanager.eventgrid.models.TopicSharedAccessKeys"
- "com.azure.resourcemanager.eventgrid.models.TopicTypeInfo"
- "com.azure.resourcemanager.eventgrid.models.TopicTypes"
- "com.azure.resourcemanager.eventgrid.models.Topics"
metadata: {}
package: "com.azure.resourcemanager.eventgrid.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventgrid:1.0.0-beta.1
