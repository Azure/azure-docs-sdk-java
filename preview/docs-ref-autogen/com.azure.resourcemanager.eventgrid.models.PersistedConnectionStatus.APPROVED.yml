### YamlMime:JavaMember
uid: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PersistedConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.eventgrid.models.PersistedConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PersistedConnectionStatus.APPROVED"
  summary: "Static value Approved for PersistedConnectionStatus."
  syntax: "public static final PersistedConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.eventgrid.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventgrid:1.0.0-beta.1
